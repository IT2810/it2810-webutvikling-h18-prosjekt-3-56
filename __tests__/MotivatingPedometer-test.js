import React from 'react';
import Pedometer from '../features/MotivatingPedometer.js';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ProgressCircle }  from 'react-native-svg-charts';


it("renders progressCircle correct", () => {
	const tree = renderer
	.create(
		<ProgressCircle 
			style = {{height: 350}}
			progress = {0.7 / 1}
			progressColor = {'black'}
		/>
	)
	.toJSON();
	expect(tree).toMatchSnapshot();
})

it("renders correctly", () => {
	const tree = renderer.create(<Pedometer />).toJSON();
	expect(tree).toMatchSnapshot();
})