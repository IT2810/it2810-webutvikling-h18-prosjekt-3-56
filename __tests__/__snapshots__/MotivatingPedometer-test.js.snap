// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
      "marginTop": "3%",
    }
  }
>
  <View
    style={
      Object {
        "height": "100%",
        "marginHorizontal": "3%",
        "width": "94%",
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "height": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column-reverse",
        "height": "90%",
        "position": "absolute",
        "width": "100%",
        "zIndex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": "40%",
          "justifyContent": "center",
          "marginBottom": "3%",
          "width": "100%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": undefined,
            "fontWeight": "400",
          }
        }
      >
           
        100 /
      </Text>
      <TextInput
        allowFontScaling={true}
        onSubmitEditing={[Function]}
        placeholder="1000"
        style={
          Object {
            "fontSize": undefined,
            "fontWeight": "400",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": "45%",
          "justifyContent": "center",
          "width": "100%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      />
    </View>
  </View>
</View>
`;

exports[`renders progressCircle correct 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "height": 350,
    }
  }
/>
`;
